README.md

```diff
@@ -103,6 +103,7 @@ that are exposed by libosdp.
 git clone https://github.com/goToMain/libosdp --recurse-submodules
 # git submodule update --init (if you missed doing --recurse-submodules earlier)
 cd libosdp
+./configure.sh
```

Comment:
This project has two build systems: cmake and make. The `configure.sh` is used for the make build while this section of the document is describing the cmake build (and is right as it is). If you wish to document the make build, please do that in a separate section.